README.md

```diff
@@ -1,37 +1,49 @@
 # Send and Receive MMS C#
-<a href="http://dev.bandwidth.com"><img src="https://s3.amazonaws.com/bwdemos/BW-VMP.png"/></a>
-</div>
+<a href="http://dev.bandwidth.com/docs/messaging/quickStart">
+  <img src="./icon-messaging.svg" title="Messaging Quick Start Guide" alt="Messaging Quick Start Guide"/>
+</a>
 
  # Table of Contents
 
 <!-- TOC -->
 
+- [Send and Receive MMS C#](#send-and-receive-mms-c)
+- [Table of Contents](#table-of-contents)
 - [Description](#description)
-- [Bandwidth](#bandwidth)
+- [Pre-Requisites](#pre-requisites)
 - [Environmental Variables](#environmental-variables)
+- [Running the Application](#running-the-application)
 - [Callback URLs](#callback-urls)
-    - [Ngrok](#ngrok)
+  - [Ngrok](#ngrok)
 
 <!-- /TOC -->
 
-# Description
-This repository contains two C# projects: SendReceiveMMS and Server. SendReceiveMMS is used to send multimedia messages from your bandwidth number to your `USER_NUMBER`. 
-If your environment variables are configured properly, simply running this project should send a text containing a photo to the number you provide.
-Server is used for handling inbound and outbound webhooks from Bandwidth. In order to use the correct endpoints, you must check the "Use multiple callback URLs" box on the application page in Dashboard. Then in Dashboard, set the INBOUND CALLBACK to `/callbacks/inbound/messaging` and the STATUS CALLBACK to `/callbacks/outbound/messaging`.
-The same can be accomplished via the Dashboard API by setting `InboundCallbackUrl` and `OutboundCallbackUrl` respectively.
+# Description  
+Using a tool capable of making POST requests such as [Postman](https://www.postman.com/), send a `POST` request to the `/sendMessage` endpoint with a json body containing the `to` and `text` fields. The `to` field should be an array of E.164 formatted phone numbers to send the SMS to.
 
-Inbound callbacks are sent notifying you of a received message on a Bandwidth number, this app prints the number of media received, if any.
-Outbound callbacks are status updates for messages sent from a Bandwidth number, this app has a dedicated response for each type of status update.
+```json
+{
+  "to": ["+19195551234"],
+  "text": "Hello World!"
+}
+```
```

Comment:
```suggestion
```http
POST /sendMessage
Host: localhost:5001
Content-Type: application/json

{
  "to": "+19195551234",
  "text": "Hello World!"
}
```
```